src/library/scala/annotation/meta/constructor.scala

```diff
@@ -0,0 +1,19 @@
+/*                     __                                               *\
+**     ________ ___   / /  ___     Scala API                            **
+**    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
+**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
+** /____/\___/_/ |_/____/_/ | |                                         **
+**                          |/                                          **
+\*                                                                      */
+package scala.annotation.meta
+
+/**
+ * An annotation marked with `@constructor` signals that an annotation can be
+ * placed at a constructor.
```

Comment:
the other annotations in the `meta` package don't specify a recommended target, but instead define the default target if there is no meta-annotation on an annotation usage.